docs/Developers-Guide.md

```diff
@@ -282,40 +228,136 @@ Redis is used as:
 <a name="bot-developer-guide"></a>
 ## Bot Developer Guide
 
-### Template
+There's a dummy bot including tests at `intelmq/tests/bots/test_dummy_bot.py`.
 
+You can always start any parser directly from command line by either invoking the script or the python module. Don't forget to give an bot id as first argument.
```

Comment:
I thiink we should mention that run with other user than intelmq can create some problems with log permissions etc...